includes/post-types.php

```diff
@@ -24,7 +24,7 @@ function give_setup_post_types() {
 
 	// Give Forms single post and archive options.
 	$give_forms_singular = give_is_setting_enabled( give_get_option( 'forms_singular' ) );
```

Comment:
@henryholtgeerts Can you define a default value to `enabled` for this?

Reply:
Done, ref: https://github.com/impress-org/givewp/commit/218c59dd5ee53a02ae3d91eb1ec96c165afc7e4d